vmdb/db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Retirement/StateMachines/Methods.class/__methods__/start_retirement.rb

```diff
@@ -0,0 +1,29 @@
+#
+# Description: This method sets the retirement_state to retiring
+#
+
+$evm.log("info", "Listing Root Object Attributes:")
+$evm.root.attributes.sort.each { |k, v| $evm.log("info", "\t#{k}: #{v}")  }
+$evm.log("info", "===========================================")
+
+vm = $evm.root['vm']
+if vm.nil?
+  $evm.log('info', "VM Object not found")
+  exit MIQ_ABORT
+end
+
+if vm.retired?
+  $evm.log('info', "VM is already retired. Aborting current State Machine.")
+  exit MIQ_ABORT
+end
+
+if vm.retiring?
+  $evm.log('info', "VM is in the process of being retired. Aborting current State Machine.")
+  exit MIQ_ABORT
```

Comment:
Use `fatal` logging messages here if we are exiting the state machine with `MIQ_ABORT`.